mne/transforms.py

```diff
@@ -26,25 +26,80 @@
 als_ras_trans_mm = als_ras_trans * [0.001, 0.001, 0.001, 1]
 
 
+_str_to_frame = dict(meg=FIFF.FIFFV_COORD_DEVICE,
+                     mri=FIFF.FIFFV_COORD_MRI,
+                     mri_voxel=FIFF.FIFFV_MNE_COORD_MRI_VOXEL,
+                     head=FIFF.FIFFV_COORD_HEAD,
+                     mni_tal=FIFF.FIFFV_MNE_COORD_MNI_TAL,
+                     ras=FIFF.FIFFV_MNE_COORD_RAS,
+                     fs_tal=FIFF.FIFFV_MNE_COORD_FS_TAL,
+                     ctf_head=FIFF.FIFFV_MNE_COORD_CTF_HEAD,
+                     ctf_meg=FIFF.FIFFV_MNE_COORD_CTF_DEVICE)
+_frame_to_str = dict((val, key) for key, val in _str_to_frame.items())
+
+_verbose_frames = {FIFF.FIFFV_COORD_UNKNOWN: 'unknown',
+                   FIFF.FIFFV_COORD_DEVICE: 'MEG device',
+                   FIFF.FIFFV_COORD_ISOTRAK: 'isotrak',
+                   FIFF.FIFFV_COORD_HPI: 'hpi',
+                   FIFF.FIFFV_COORD_HEAD: 'head',
+                   FIFF.FIFFV_COORD_MRI: 'MRI (surface RAS)',
+                   FIFF.FIFFV_MNE_COORD_MRI_VOXEL: 'MRI voxel',
+                   FIFF.FIFFV_COORD_MRI_SLICE: 'MRI slice',
+                   FIFF.FIFFV_COORD_MRI_DISPLAY: 'MRI display',
+                   FIFF.FIFFV_MNE_COORD_CTF_DEVICE: 'CTF MEG device',
```

Comment:
I guess the next line makes it clearer it could be any. I think the idea is that it would be any non-NM system. I'll modify this one to be clearer even though it deviates from what one would see with C.